src/main/java/seedu/address/model/person/Appointment.java

```diff
@@ -0,0 +1,146 @@
+package seedu.address.model.person;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.stream.Collectors.toList;
+
+/**
+ * Appointment class to store appointment objects to represent tutee and tutor relations
+ */
+public class Appointment {
+
+    private final Email email;
+    private final String subject;
+    private final LocalDateTime dateTime;
+    private final String location;
+
+    private final static String formatter = "dd MM yyyy";
+
+    private static List<Appointment> appointmentList = new ArrayList<>();
```

Comment:
I don't think the variable `appointmentList` and its respective methods are required in `Appointment` class, since Yuting is handling the `List` classes, which already have a `AppointmentList`.